### ISSUE TITLE
www.google.com Parse error: HPE_INVALID_CHUNK_SIZE Invalid character in chunk size

### ISSUE THREAD

**Issue by tpisto** [nodejs/llhttp#17]
First of all, thank you for this great effort to solve the very hard http parsing problem!

I tried multiple websites, and most of them are working fine. But for some reason when trying to parse www.google.com the parser says "Parse error: HPE_INVALID_CHUNK_SIZE Invalid character in chunk size"

With same data the http_parser parses it fine, so seems it's not problem with the data.

**Comment by indutny** [comment]
@tpisto thank you for kind words!

Is there a test case that I you would suggest me to try?

**Comment by indutny** [comment]
As a matter of fact, `node --http-parser=llhttp -e "https.get('https://www.google.com', (res) => res.pipe(process.stdout))"` appears to be working.

**Comment by tpisto** [comment]
Intersting indeed - I'll set up an sweet and short example repo tomorrow and investigate this further.

**Comment by indutny** [comment]
I'm glad to hear that the issue is resolved, but would you care to share the details for posterity?

**Comment by tpisto** [comment]
Sure, and sorry for bothering you. It was interesting case indeed, and only happened in my tests for google.com in the last chunk. The reason was simple. 

What I did with http_parser:
response -> char string -> parser

When testing llhttp I had changed the process also, so it went
response -> v8 utf8 char string -> parser

So in the majority of cases it worked just fine, but in the google.com response the last chunk lead to error. That was just my bad, and again sorry for bothering you about it.
